src/Core/src/Handlers/Image/ImageHandler.Android.cs

```diff
@@ -21,9 +20,14 @@ protected override ImageView CreatePlatformView()
 			return imageView;
 		}
 
+		protected override void ConnectHandler(ImageView platformView)
+		{
+			platformView.ViewAttachedToWindow += OnPlatformViewAttachedToWindow;
+		}
+
 		protected override void DisconnectHandler(ImageView platformView)
 		{
-			base.DisconnectHandler(platformView);
```

Comment:
Did we remove `base.DisconnectHandler()` on purpose?

Reply:
Base method is empty and many other handlers do not call the base method.
Anyway, I'll bring it back just in case someone in the future wants to fix the situation in general.